src/SAWScript/Prover/MRSolver/Monad.hs

```diff
@@ -831,6 +861,28 @@ instantiateFunAssump fassump =
      rhs <- substTermLike 0 evars $ fassumpRHS fassump
      return (args, rhs)
 
+-- FIXME HERE NOW: delete this and remove the preconditions from the env
+-- | Look up the precondition associated with a function name, if there is one
+-- mrLookupPrecond :: FunName -> MRM (Maybe Term)
+-- mrLookupPrecond nm = Map.lookup nm <$> mrePreconditions <$> mriEnv <$> ask
+
```

Comment:
As with the comment above, looks like this is done, so it should probably be removed?

Reply:
Yep